sdks/go/container/build.gradle

```diff
@@ -47,12 +47,14 @@ golang {
   }
 }
 
+def defaultTag = project.rootProject.hasProperty(["docker-tag"]) ? [project.rootProject["docker-tag"], 'latest'] : ['latest']
```

Comment:
By default this will always push latest tag, which I'm not sure we actually want to force. Would it be better to make docker-tag a comma-separated string list parameter and use split? That way we can also allow people to push more than one tag that isn't just latest.

Reply:
Thanks for the comment.

> By default this will always push latest tag, which I'm not sure we actually want to force.

FYI, 'latest' tag is only attached when we use `dockerTagPush` command (not `dockerPush`).

If we allow a comma-separated string in `docker-tag`, it will break `docker` task. `docker` and `dockerTag` are different commands where `docker` command only works with the tag specified by `docker-tag` property but `dockerTag` command respects the list of tags in `tags`.

We should define a separate property something like `docker-aux-tag` (or `docker-secondary-tag`) and set `docker-tag` + `docker-aux-tag` for `tags` to let users append their own tags. Do you want to go with this option?

Reply:
ah, I see. Would it work to just ignore docker-tag for dockerTagPush and just use one flag like docker-tags or docker-tag-list? Mostly I don't quite like the idea of splitting the tag flag - if docker-tags isn't given, we can just construct a list from docker-tag, or use `["latest"]`?

Reply:
Changed to getting from 'docker-tag-list' and generating the default value from 'docker-tag' if the first option is not available.